python/cucim/src/cucim/core/operations/spatial/tests/test_random_flip.py

```diff
@@ -49,7 +49,10 @@ def test_rand_flip_batchinput():
     arr = get_input_arr()
     flip_arr = get_flipped_data()
     arr_batch = np.stack((arr,) * 8, axis=0)
-    np_output = spt.rand_image_flip(arr_batch, prob=1.0, spatial_axis=(2, 3))
+    np_output = spt.rand_image_flip(arr_batch,
+                                    prob=1.0,
+                                    spatial_axis=(2, 3),
```

Comment:
Parameter input is not consistent. Here it uses `(2, 3)`, but below code, it uses `[2, 3]`.